From 957e394aa4e011e98cd1dfc09f97960bc94b1697 Mon Sep 17 00:00:00 2001
From: Ileana Dumitrescu <ileanadumitrescu95@gmail.com>
Date: Tue, 7 Jan 2025 17:07:53 +0200
Subject: [PATCH] demo.at: Skip test if deplibs_check_method is not pass_all

The test 'deplibs_check_method' was designed to verify pass_all
works as expected. Many warnings occur for systems that do not
have deplibs_check_method set to pass_all, so this test is skipped
on those.

* tests/demo.at: Add check for deplibs_check_method=pass_all.
---
 tests/demo.at | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/demo.at b/tests/demo.at
index 0fff438a..036e434e 100644
--- a/tests/demo.at
+++ b/tests/demo.at
@@ -434,6 +434,9 @@ AT_CLEANUP
 
 AT_SETUP([deplibs_check_method])
 
+# Skip test if deplibs_check_method is not pass_all
+AT_CHECK([$LIBTOOL --config | $EGREP '^deplibs_check_method="pass_all' || exit 77], [], [ignore], [ignore])
+
 _LT_DEMO_SETUP
 
 AT_DATA([demo.mk],
-- 
2.45.2

